.travis.yml

```diff
@@ -34,6 +34,5 @@ before_cache:
 cache:
   directories:
     - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
```

Comment:
Do you have more context on why this works? Because:
- the focus team hasn't run into issues
- The official docs recommend putting this in here: https://docs.travis-ci.com/user/languages/android/#caching

I wonder if our cache some how got messed up and we actually just need to purge it (here: https://travis-ci.org/mozilla-mobile/firefox-tv/caches ?) and this change actually is otherwise not necessary.

Reply:
Here's a bit on clearing caches: https://docs.travis-ci.com/user/caching/#clearing-caches

"Sometimes you spoil your cache by storing bad data in one of the cached directories, or your cache can become invalid when language runtimes change."